fe/src/stores/useStudentStore.ts

```diff
@@ -115,14 +111,15 @@ export const useStudentStore = defineStore('user/student', {
      * Load a student by its id
      * @param studentId the id of the student
      */
-    async loadStudent(studentId: number) {
+    async loadStudent(studentId: number, fail: Function) {
       this.isLoading = true
 
       await instance.get(`students/${studentId}/`).then(async ({ data }) => {
         await this.transformStudent(data)
 
         this.currentStudent = new Student(data as Student)
       })
+          .catch(() => fail())
```

Comment:
I think this can be written as .catch(fail)

Reply:
I get this typescript error when doing this: 
`TS2345: Argument of type 'Function' is not assignable to parameter of type '(reason: any) => PromiseLike<never>'.   Type 'Function' provides no match for the signature '(reason: any): PromiseLike<never>'.`